mne/decoding/time_gen.py

```diff
@@ -359,7 +354,7 @@ def predict(self, epochs, test_times=None):
                                  for t_test in t_train]
                                  for t_train in test_times_['slices']]
         # Store all testing times parameters
-        self.test_times_ = test_times_
+        self.test_times = test_times_
```

Comment:
@agramfort what about dictionaries that are passed as params, but one key is updated: shall I keep it as
`my_dict[param_key]` `my_dict[attribute_key_]`  or `my_dict_[key`]